servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,40 @@
-- Repeat the process of considering the business relevance, system impact and load testing with each [Custom Check](/servicecontrol/plugins/custom-checks.md).
-- Repeat the process of considering the business relevance, system impact and load testing with each Endpoint [Audit](/nservicebus/operations/auditing.md).
+When configuring ServiceControl for a new environment follow these steps:
 
-##### Tips and Tricks
+- Read the [Capacity Planning](/servicecontrol/capacity-and-planning.md) and [Troubleshooting](/servicecontrol/troubleshooting.md) guides for ServiceControl.
+- Turn off Auditing and remove ServiceControl plugins from Endpoints in the new environment.
+- Run a performance test using the expected peak and average message throughput for the environment to baseline the system. This test suite should include simulating infrastructure outages that will cause large numbers of message processing failures. 
+- For each Endpoint, install and configure the Heartbeat plugin (if needed). Re-run the performance test suite and monitor ServiceControl to ensure that it is able to effectively monitor the system under load. This may require adjustments to the Heartbeat Interval. Re-run the performance tests after each adjustment.
+- For each endpoint, turn on Auditing (if needed) and re-run the performance tests to determine impact.
+- For each endpoint, turn on any required Custom Checks and re-run the performance tests to determine impact.
 
-- If running anti-virus software exclude the ServiceControl [database directory](/servicecontrol/configure-ravendb-location.md) from virus checks.
-- The recommended approach is that customers don't downgrade major and minor releases of ServiceControl. ServiceControl uses an embedded database and changes to the internal data structures can occur between releases. Rolling back may cause index corruption or data loss. The recommended approach is that customers perform testing in a lower environment before upgrading in production environments.
\ No newline at end of file
+NOTE: There are some factors that influence ServiceControl resource usage and performance that are less likely to change between environments but are specific to the system being monitored such as choice of transport and average message size. 
+
+
+## Specific Considerations
+
+- Each environment should have a dedicated ServiceControl instance.
+- In production, ServiceControl should run on a **dedicated machine**.
+- Turn off [Message Auditing](/nservicebus/operations/auditing.md) if it is not needed. The audit ingestion capability of ServiceControl is primarily to support system analysis with ServiceInsight. If ServiceInsight is not deployed in an environment then configure ServiceControl to read audit messages from an empty queue or turn off messaging auditing for each endpoint. Message auditing may be important for some endpoints but not other others. 
```

Comment:
@weralabaj I don't think we are trying to make a case for the value of auditing. We are filling a gap in the documentation around performance considerations and the very abrupt limits that you might hit with SC if its deployed without care. It's advice akin to 'keep your tires properly inflated and you will improve your car's fuel consumption' rather than 'everyone needs a Ferrari'